src/Tools/babylon.assetsManager.ts

```diff
@@ -669,6 +669,71 @@ module BABYLON {
         }
     }
 
+    /**
+     * Define a task used by {BABYLON.AssetsManager} to load atlas texture
+     */

```

Comment:
Actually I thought about this one and we could just use TextFileAssetTask directly
This one is too much specific and only useful for image atlas